snooty/parser.py

```diff
@@ -35,6 +35,10 @@
     BuildIdentifierSet,
 )
 
+# XXX: Work around to get snooty working with Python 3.8 until we can fix
```

Comment:
Is there a convention around `XXX`? This seems kind of like a TODO.

Reply:
XXX indicates something that's gross or a bug; basically the same as FIXME

https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment